### PR TITLE
silence PHP 8.1 deprecations

### PR DESCRIPTION
**Pull request by 3kbest**
Add ReturnTypeWillChange attribute to silence PHP 8.1 deprecations

### CODE REVIEW

File: src/Curl/CaseInsensitiveArray.php [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 namespace Curl;
 
+use ReturnTypeWillChange;
```
Comment (zachborboa) [inline src/Curl/CaseInsensitiveArray.php:5]: I think this line can be removed

**Review by zachborboa — CHANGES_REQUESTED** [review]
`use ReturnTypeWillChange;` should be removed, but otherwise LGTM

File: src/Curl/CaseInsensitiveArray.php (hunk 1 above)
Comment (3kbest) [reply src/Curl/CaseInsensitiveArray.php:5]: Then `#[\ReturnTypeWillChange]` need to be used instead of `#[ReturnTypeWillChange]`. 

What is preferred?

Comment (3kbest) [reply src/Curl/CaseInsensitiveArray.php]: Removed it now

**Review by zachborboa — APPROVED** [review]

**Comment by zachborboa** [comment]
Thanks!

**Pull request merged**